### ISSUE TITLE
Windows 1607 Dokan Driver

### ISSUE THREAD

**Issue by BranicYeti** [keybase/client#4112]
I created an [issue at keybase-issues](https://github.com/keybase/keybase-issues/issues/2530) for this problem, but haven't heard anything since. I wonder if I perhaps posted it in the wrong place, so I'm adding it here in an attempt at getting Keybase working for me again as soon as possible.

---

I was having an issue where a fresh install of Windows 1607 (with Bash on Ubuntu on Windows enabled) and a fresh install of Keybase would not show the K drive that comes with KBFS. `keybase status` suggested that KBFS was running, but the status log was saying that the "write journal was not enabled". I wondered if the Linux Subsystem was causing an issue, so freshly installed Windows again and didn't enable LXSS before installing Keybase. This time, I saw an error message that before I either missed or didn't get.
![dokan-driver-error](https://cloud.githubusercontent.com/assets/1971818/18038504/f59c4b64-6d90-11e6-9a2d-323382f8cf5b.jpg)
I assume this error is the cause of my earlier issues. A quick skim of the log file showed the same error message:
```
2016-08-29T02:35:43.474905 - [WARN kbfs asm_386.s:1585] 021 KBFS Status failed: Write journal not enabled
2016-08-29T02:35:44.678158 - [ERRO kbfs mounter.go:53] 022 Failed to mount dokan filesystem (i=16): Dokan failed: -3
2016-08-29T02:35:47.897250 - [ERRO kbfs mounter.go:53] 023 Failed to mount dokan filesystem (i=32): Dokan failed: -3
2016-08-29T02:35:54.304184 - [ERRO kbfs mounter.go:53] 024 Failed to mount dokan filesystem (i=64): Dokan failed: -3
```
I'll hold off doing what Windows suggests and installing Dokan myself, since I believe I saw advice in the past against such a thing, and I haven't really looked since to see if there is still an issue with running your own Dokan library, unless you can tell me otherwise or suggest a fix I can do before the next release of the Keybase app.

**Comment by gabriel** [comment]
@zanderz

**Comment by zanderz** [comment]
I have spent a bunch of time installing fresh Windows AU and testing, but I can't get this blocked driver popup. Separately, I submitted a package for Microsoft driver signing, but 8 days later it was still pending, I plan to try again when I get a chance.

Here is the discussion in the Dokany dev team about getting past this driver signing - they haven't completed it yet:
https://github.com/dokan-dev/dokany/issues/271#issuecomment-241187792

**Comment by BranicYeti** [comment]
Just in case I messed something myself, I just did another fresh install and got the same results. Not sure if they help, or differ to your own testing, but the steps I took are as follows:

* Fresh install of Windows 10 Pro using a USB drive created with Windows Media Creation Tool.
* Install all updates because the install doesn't come with 1607 already. A few drivers got installed along the way too (not my favourite Windows feature) but nothing by me.
* After all updates, I restarted again for good measure.
* Install Keybase for Windows from the downloads page. I did try the testing version from prerelease.keybase a few installs ago, but never saw the popup window so haven't tried it since, and I figured it would be best to stick to the production branch.
* Get to the point where Keybase says a restart is required and opens the app ready for log in.
* I focused both windows, so not sure which one triggered the popup, but a few seconds later, it appeared.

Just looking at that comment you linked, I do have Secure Boot enabled, but it's not necessary. I'll try installing again with it disabled and see if I have any luck.
